src/showPokemonUseEffect.js

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react"
+
+export function ShowPokemonUseEffect({pokeName}){
+
+  const [allPokemon, setAllPokemon] = useState([])
+  const PokeURL = `https://pokeapi.co/api/v2/pokemon/${pokeName}`
+
+  useEffect(()=>{
+    fetch(PokeURL)
+    .then(response => response.json())
+    .then(data => {
+      setAllPokemon(currentList => [...currentList, data])
```

Comment:
Why are you adding some data to the array? How many pokemons would you like to show here?